spec/cf_app_discovery/app_info_configurer_spec.rb

```diff
@@ -96,21 +117,32 @@ def routes(path)
     context "when resources field is nil" do
       it "sets a route on the input resource" do
```

Comment:
Potentially the test names could be updated to remove use of the word `route`

Reply:
good idea, will do